jav321.py

```diff
@@ -0,0 +1,70 @@
+import requests
+from bs4 import BeautifulSoup
+from lxml import html
+
+
+def main(number: str) -> dict:
+    result = requests.post("https://www.jav321.com/search", data={"sn": number})
```

Comment:
this change does not support the proxy from `config.ini `

you may want to take a look at https://github.com/yoshiko2/AV_Data_Capture/blob/master/ADC_function.py#L100  and do something similar

Reply:
forgot about proxy...